nncf/openvino/pot/engine.py

```diff
@@ -100,7 +100,7 @@ def calc_per_sample_metrics(
 ) -> List[Dict[str, Any]]:
     per_sample_metrics = []
     for inputs in dataset.get_data(subset_indices):
-        value = val_func(compiled_model, [inputs])
+        value, _ = val_func(compiled_model, [inputs])
```

Comment:
Could you share the test plan for changes related to wrapper for POT quantization with accuracy control?

Reply:
Build # 9949